src_c/rwobject.c

```diff
@@ -31,6 +31,15 @@
 
 #include "doc/pygame_doc.h"
 
+#if defined(_WIN32)
+#define PG_LSEEK _lseeki64
+#elif defined(__APPLE__)
+/* Mac does not implement lseek64 */
+#define PG_LSEEK lseek
+#else
+#define PG_LSEEK lseek64
+#endif
+
```

Comment:
Proposed https://github.com/pygame/pygame/pull/3898 to improve this on non-glibc platforms.